degenotate_lib/vcf.py

```diff
@@ -42,6 +42,12 @@ def read(globs):
     # A quick check to make sure all provided outgroup species are in the VCF file
     ## Outgroups
 
+    if 'AA' in globs['vcf'].header.info:
+        globs['vcf-polarized'] = True
+        CORE.printWrite(globs['logfilename'], 1, "# WARNING: " + "provided VCF appears to be polarized (has ancestral allele field in the header) => will try to recalculate derived allele frequency and run imputed MKT framework.");
```

Comment:
Maybe I'm getting this wrong, but if the warning is here with `1` as the verbosity option to `printWrite()`, it will be printed to the screen and mess up the status report later on. (I think the options for `printWrite()` are kind of messed up now that I'm looking at it, but that's another issue.) If we want to not mess up the status report in the shell, we should just have a flag here that is returned to [degenotate.py](https://github.com/harvardinformatics/degenotate/blob/4cf3758b7d7434682fbb53a41588390a83e853f7/degenotate.py#L78) and [printed out after the status update on the next line](https://github.com/harvardinformatics/degenotate/blob/4cf3758b7d7434682fbb53a41588390a83e853f7/degenotate.py#L79).